README.md

```diff
@@ -10,14 +10,16 @@ Copyright Artefactual Systems Inc (2020).
 
 ## Installation
 
+Below are the developer quickstart instructions. [Production installation](#production-installation) instructions are provided at the bottom of this page.
+
```

Comment:
This might be the only line to go over 80 characters? Maybe a few of the others added below? They should be 80 max per the [style guidelines](https://github.com/artefactual/archivematica-docs/blob/12da09e3664025169450ac8b913639ff85ebba84/style_guide.md#formatting-and-punctuation). 

Have a look at the links below for how to bring them in-line with the using reference style links. For this line I think you need:

Example: 
```markdown
Below are the developer quickstart instructions. 
[Production installation][prod-install-1] instructions are provided at the 
bottom of this page.

...

[prod-install-1]: #production-installation
```
And I think you only need it where you can't get text+link into 80 characters by splitting the line.